types/applications/forms/setupConfigurationForm.d.ts

```diff
@@ -1,3 +1,3 @@
 declare class SetupConfigurationForm extends FormApplication {
-  _updateObject(event?: any, formData?: object): void;
+  _updateObject(event?: Event, formData?: object): Promise<any>;
```

Comment:
Might be worth marking these `Promise<any>`s as TODO to make them easier to find.

Reply:
will do so

Reply:
done